aten/src/ATen/native/cpu/VectorizedCopy.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
```

Comment:
So actually the compilation unit(s) that this is included in need to be in `native/cpu`, not just this header. Probably need to split out the implementation from TensorShape.cpp and use dynamic dispatch

Reply:
Actually, I will just remove this as exposed API and integrate it with repeat.

Reply:
Is there a reason all of this lives in a header?

Reply:
@cpuhrsch Better code refactoring? I am open to suggestions.

Reply:
We've been writings these kernels within .cpp files. Take a look at UnaryKernels.cpp. This also includes information on how to setup the dispatch etc.

Reply:
@cpuhrsch, I am gonna move this file out of cpu folder and into native for now. I will multiple dispatch option thin in a separate PR.